tools/make/tidy.toml

```diff
@@ -265,3 +265,44 @@ description = "Run the dependency checker"
 category = "ICU4X Development"
 command = "cargo"
 args = ["run", "--manifest-path", "tools/make/depcheck/Cargo.toml"]
+
+[tasks.codegen]
+description = "Run the codegen tool"
+category = "ICU4X Development"
+command = "cargo"
+args = ["run", "--manifest-path", "tools/make/codegen/Cargo.toml"]
+
+[tasks.codegen-fmt]
+description = "Run the codegen tool and format the output"
+category = "ICU4X Development"
+dependencies = [
+    "codegen"
+]
+command = "cargo"
+args = ["fmt"]
```

Comment:
this formats the whole repo, which is irreversible and it shouldn't do

Reply:
that's why I made separate `cargo make codegen` and `cargo make codegen-fmt` jobs.

Reply:
@Manishearth Can I run `rustfmt` programatically in the codegen tool, like after I run Askama, then I run Rustfmt on the output?

Reply:
the tidy CI job now fails if any file in the repo is unformatted, which is already what the fmt CI job checks

Reply:
I need to run `rustfmt` on the askama output, which is not yet formatted, in order to check with `git` whether there is a diff on the generated code.

Reply:
You can run rustfmt on a single file instead of the whole repo

Reply:
I don't really want to put the list of generated files into this tidy script. I'd be fine if running the codegen tool ran the formatter automatically.

Reply:
Why even bother formatting them? Just put `// @generated` in the first line.

Reply:
@sffc Yeah the `codegen` task should just run `rustfmt foo.rs` on every output file. We don't need a separate task

Reply:
I need an issue to track this. https://github.com/unicode-org/icu4x/issues/6327